interactive/run.py

```diff
@@ -119,13 +129,13 @@ def write_files(checkout, analysis_request, codelist):
 
 
 def commit_and_push(checkout, analysis_request):
-    msg = f"Interactive analysis of codelist {analysis_request.codelist} ({analysis_request.id})"
+    msg = f"{analysis_request.id}: codelist {analysis_request.codelist} by {analysis_request.user.email}"
```

Comment:
Hmm, this exposes the users email directly in the web view. Probably shouldn't do that.

Reply:
I've tweaked the commit message again, to remove the users email, and but the id back at the end as having it first was ugly.

Reply:
I've just raised this as a question in slack. I think it's really important that we have a public audit log, but I agree that having the user's email in there is potentially a problem

Reply:
Brian seems to think it's reasonable. I've created a ticket so we remember to make users aware https://github.com/opensafely-core/interactive.opensafely.org/issues/135

Reply:
For clarity, their email address is visible in the actual git commit history. It's just not directly visible the github web UI (to avoid easy scraping, I assume). I think this is a happy middle ground?

Reply:
But make users aware their email will be a matter of public record is a good thing